App refresh

sv101 opened the pull request:
Please review the work and let me know if you have any comments or questions!

sv101 committed:
Create app.R

sv101 committed:
initial changes

sv101 committed:
changes

sv101 committed:
Update app.R

sv101 committed:
the app is now updated

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Additional updates

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
updates

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Update app.R

sv101 committed:
Reorganization

sv101 committed:
reorganization #2

sv101 committed:
reorganization 3

sv101 committed:
reload of README

sv101 committed:
Update app.R

Added reactive checkboxes

Xinyue-Tang reviewed (commented):
Hi! In the overview page, the date and month is 00/00/2022. Also, in the challenge page, I think it is better to use the symbol image to check answers, like the green right and red wrong  symbol.

hy2021-IT commented:
Whenever I try to launch ui or server, an package called CombMSC is required but there is no source to install package.

sv101 committed:
Update app.R

sv101 commented:
> Whenever I try to launch ui or server, an package called CombMSC is required but there is no source to install package.

You're looking at the wrong branch. The branch that I am trying to merge is `SU22_update`

hy2021-IT reviewed (dismissed):
Besides those that have been mentioned above, the app is ok to proceed.

Xinyue-Tang reviewed (changes requested):
Hi! I did not see the changes! In the overview page, the date and month is 00/00/2022. Also, in the challenge page, I think it is better to use the symbol image to check answers, like the green right and red wrong symbol.

sv101 committed:
Update app.R

sv101 commented:
> Hi! I did not see the changes! In the overview page, the date and month is 00/00/2022. Also, in the challenge page, I think it is better to use the symbol image to check answers, like the green right and red wrong symbol.

Thank you for your review.
I rectified the date issue. 
The Challenge page has functionality that can help the user without visual cues.

sv101 committed:
Update app.R

luqije reviewed (changes requested):
- Missing DESCRIPTION file which may be the reason caused by "Unable to cite an app, please check warnings" for "Cite this app as:". 
- For the dataset, GDP growth rate of U.S., the plot with the trend has no change after adding seasonal. 
- Missing the "New Process" button for the multiple processes on the Explore page.  
- Missing alt for the plots.
- Missing references on the shinyjs and zoo packages.
- The information and feedback buttons on the top right are not working properly.
- When testing how the information is displayed as the window size changes, the question tends to overlap on the plot on the Challenge page.   
 Suggestion: 
- In the Prerequisites, add explanations for the components so the users have a better understanding of what to look for.
- In the Example, adding feedback to explain different plots with different components might be better.  
- The feedback in Challenge should be displayed more formally instead of looking like R output.

Commit:
update

added DESCRIPTION file and update (issue in the Challenge page)

neilhatfield committed:
Neil's Changes

Neil's changes to address the reported problem about the hint showing up too early as well as the scoring icon remaining

Commit:
update app.R

neilhatfield committed:
Address reported issues

This update addressed the issue with the missing challenge page images and Description file issue. See Proposed Change note in app.R

Commit:
update the app.R

created a questionBank file

Commit:
update the questionBank

neilhatfield committed:
Challenge Page Updates

Commit:
update

Commit:
update

Commit:
update on simulation

Commit:
update simulation

Commit:
update

Commit:
update simulation
